xlsx.cabal

```diff
@@ -27,7 +27,7 @@ Maintainer:          qrilka@gmail.com
 
 Category:            Codec
 Build-type:          Simple
-Tested-with:         GHC == 8.8.4, GHC == 8.10.4, GHC == 9.0.2, GHC == 9.2.2
+Tested-with:         GHC == 8.6.4, GHC == 8.8.4, GHC == 8.10.4, GHC == 9.0.2, GHC == 9.2.2
```

Comment:
Where id 8.6 come from? The list here is the list from Github actions and 8.6 was previously explicitly excluded

Reply:
As it's the version I'm using I thought it'd be useful to add it, I didn't know this field had automatic content or that 8.6 was excluded 😅

Reply:
It's not automatic but the idea is to have there supported versions which are tested in CI

Reply:
understood :)